doc/admin-man/security.tex

```diff
@@ -1445,6 +1446,11 @@ \subsubsection{\label{sec:NTSSPI-Authentication}Windows Authentication}
 computers. 
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsubsection{\label{sec:MUNGE-Authentication}Ask MUNGE for Authentication}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+Ask the MUNGE service to validate both sides of the authentication.
```

Comment:
Could you include a link to the MUNGE documentation page?  Seems to be https://dun.github.io/munge/ ?